maven-core/src/test/java/org/apache/maven/lifecycle/internal/LifecycleExecutionPlanCalculatorTest.java

```diff
@@ -63,6 +64,7 @@ public void testCalculateExecutionPlanWithGoalTasks()
     // Maybe also make one with LifeCycleTasks
 
     public static LifecycleExecutionPlanCalculator createExecutionPlaceCalculator( MojoDescriptorCreator mojoDescriptorCreator )
+            throws ComponentLookupException
```

Comment:
Who throws this?

Reply:
`DefaultLifecyclesStub#createDefaultLifecycles` does, because it mocks `PlexusContainer#lookupMap`. Both classes are test classes.